src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
```

Comment:
You could also not create a new object, but change the old one, because the task asks to "repair" it, not create another

Reply:
> You could also not create a new object, but change the old one, because the task asks to "repair" it, not create another

![image](https://github.com/mate-academy/js_inverse_robot/assets/59209337/9d5c9ef2-a59e-4a32-8188-0f4aec10a28c)